docs/source/customizations/coprocessor.mdx

```diff
@@ -1021,14 +1021,14 @@ Subsequent response chunks omit the `headers` and `statusCode` fields:
 
 ## Adding authorization claims via coprocessor
 
-To use the [authorization directives](../configuration/authorization#authorization-directives), a request needs to include **claims**—the details of its authentication and scope. The most straightforward way to add claims is with [JWT authentication](../configuration/./authn-jwt). You can also add claims with a [`RouterService` coprocessor](#how-it-works) since it hooks into the request lifecycle directly after the router has received a client request.
+To use the [authorization directives](../configuration/authorization#authorization-directives), a request needs to include **claims**—the details of its authentication and scope. The most straightforward way to add claims is with [JWT authentication](../configuration/./authn-jwt). You can also add claims with a [`RouterService` or `SupergraphService` coprocessor](#how-it-works) since it hooks into the request lifecycle before the router applies authorization logic.
 
-The router configuration needs to include at least these settings:
+The router configuration necessary to call the coprocessor for authorization claims looks like this:
```

Comment:
```suggestion
The router configuration needs to call the coprocessor for authorization claims that look like this:
```

Reply:
Hmm your suggestion doesn't quite make sense to me. I agree that my sentence isn't great either tho

Reply:
How about:
```
When processing authorization claims, the required router configuration looks like this:
```

Reply:
```suggestion
An example configuration of the router calling a coprocessor for authorization claims:
```